spec/models/salary_per_year_spec.rb

```diff
@@ -25,6 +25,15 @@
       end
     end
 
+    context 'when monthly frequency' do
+      let(:salary) { 1000 }
+      let(:salary_frequency) { 'month' }
+
+      it 'returns salary multiplied by 12' do
+        expect(convert).to eq(12_000)
+      end
+    end
+
     context 'when inexistent frequency' do
       let(:salary) { 1000 }
       let(:salary_frequency) { 'inexistent' }
```

Comment:
Is this not part of this PR though, ¬¬'. But I will change since it is an easy fix.